src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -572,7 +572,7 @@ protected function get_cursor_for_node( $id ) {
 	}
 
 	/**
-	 * get_edges
+	 * Get_edges
```

Comment:
what's up with all these changes to ucfirst?

Reply:
Actually, we can probably just get rid of the instances of method names at the top of the docblock, eh?

Reply:
we can follow up later and remove these